README.md

```diff
@@ -39,7 +39,7 @@ from aws_embedded_metrics import metric_scope
 @metric_scope
 def my_handler(metrics):
     metrics.put_dimensions({"Foo": "Bar"})
-    metrics.put_metric("ProcessingLatency", 100, "Milliseconds")
+    metrics.put_metric("ProcessingLatency", 100, "Milliseconds", 60)
```

Comment:
Can't we pass Enum value?

Reply:
Done!

Reply:
My bad, resolved it now. Thanks! Also added extra information regarding storage resolution.